src/utils/db/car_stream.rs

```diff
@@ -6,7 +6,9 @@ use cid::{
     multihash::{Code, MultihashDigest},
     Cid,
 };
-use futures::{Stream, StreamExt};
+use flume::r#async::RecvStream;
+use futures::{sink::Sink, Stream, StreamExt};
+use fvm_ipld_car::CarHeader as FvmCarHeader;
```

Comment:
I know this is a draft, but note that you cannot use `fvm_ipld_car`. The entire point is to _not_ use that library.